tunnistamo/urls.py

```diff
@@ -51,7 +52,8 @@ def get_schema(self, *args, **kwargs):
 router.register('service', ServiceViewSet)
 router.register('user_consent', UserConsentViewSet)
 
-v1_api_path = path('v1/', include((router.urls, 'v1')))
+v1_scope_path = path('scope/', ScopeListView.as_view(), name='scope-list')
```

Comment:
Why is this not under v1?

Reply:
Indeed. You can conveniently include API endpoints from different Django apps under the same router like this: https://github.com/City-of-Helsinki/aplans/blob/master/aplans/urls.py#L26

Reply:
> Why is this not under v1?

It is under v1, see next line `v1_api_path = path('v1/', include((router.urls + [v1_scope_path], 'v1')))`.

> Indeed. You can conveniently include API endpoints from different Django apps under the same router like this: https://github.com/City-of-Helsinki/aplans/blob/master/aplans/urls.py#L26

Sure, but you can only register a `ViewSet` to a router, not an `ApiView` which is the case here.